packages/webapp/webpack.config.js

```diff
@@ -4,31 +4,97 @@ module.exports = {
   entry: './src/index.ts',
   output: {
     path: path.resolve(__dirname, '..', '..', 'nbdime', 'webapp', 'static'),
-    filename: "nbdime.js",
-    publicPath: "./static/"
+    filename: 'nbdime.js',
+    publicPath: './static/',
   },
   bail: true,
   devtool: 'source-map',
   module: {
     rules: [
-      { test: /\.css$/, use: ['style-loader', 'css-loader'] },
-      { test: /\.ipynb$/, type: 'json' },
-      { test: /\.ts$/, loader: 'ts-loader' },
-      { test: /\.js$/, loader: "source-map-loader" },
-      { test: /\.html$/, loader: 'file-loader' },
-      // jquery-ui loads some images
-      { test: /\.(jpg|png|gif)$/, loader: 'file-loader' },
-      // required to load font-awesome
-      { test: /\.woff2(\?v=\d+\.\d+\.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=application/font-woff' },
-      { test: /\.woff(\?v=\d+\.\d+\.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=application/font-woff' },
-      { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=application/octet-stream' },
-      { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file-loader' },
-      { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=image/svg+xml' }
+      {
+        test: /\.css$/,
+        use: [require.resolve('style-loader'), require.resolve('css-loader')],
```

Comment:
webpack will always resolve loaders relative to itself, rather than relative to the config file — better to resolve based on the config files — as that is where our deps are defined.